docs/installation.md

```diff
@@ -120,20 +104,20 @@ Woohoo! Your OpenAPI definition is valid. 🎉
 To give a Docker container access to your OpenAPI definition files, you need to mount the containing directory as a volume. Assuming the definition is rooted in the current working directory, the command to use is:
 
 ```shell Example with lint command
-docker run --rm -v $PWD:/spec redocly/openapi-cli lint petstore.yaml
+docker run --rm -v $PWD:/spec redocly/cli lint petstore.yaml
```

Comment:
I believe this change depends on this issue: https://github.com/Redocly/redocly-cli/issues/910
@RomanHotsiy shouldn't we firstly create the corresponding repo on the Docker Hub?

Reply:
Yes, I made a mistake here by changing the docker instructions.